packages/ubuntu_desktop_installer/lib/l10n/app_es.arb

```diff
@@ -0,0 +1,113 @@
+{
+  "@@locale": "es",
+
+  "languageName": "Español",
+
+  "appTitle": "Instalador de Ubuntu para escritorio",
+
+  "backButtonText": "Volver",
+  "continueButtonText": "Continuar",
+  "restartButtonText": "Reiniciar",
+
+  "welcome": "Bienvenido",
+
+  "tryOrInstallPageTitle": "Probar o instalar",
+  "repairInstallation": "Reparar instalación",
+  "repairInstallationDescription": "Al reparar se reinstalarán todos los programas instalados sin tocar los documentos ni la configuración.",
+  "tryUbuntu": "Probar Ubuntu",
+  "tryUbuntuDescription": "Puede probar Ubuntu sin hacer ningún cambio en su equipo.",
+  "installUbuntu": "Instalar Ubuntu",
+  "installUbuntuDescription": "Instalar Ubuntu junto con (o reemplazando) su sistema operativo actual. No debería tardar mucho.",
+  "releaseNotesLabel": "Aquí puede leer las <a href=\"{url}\">notas de lanzamiento</a>.",
+  "@releaseNotesLabel": {
+    "type": "text",
+    "placeholders": {
+      "url": {}
+    }
+  },
+
+  "turnOffRST": "Apagar RST",
```

Comment:
I would have said "Desactivar RST". What do you think?

Reply:
I totally agree.